src/hooks/useWalletCloudBackup.js

```diff
@@ -8,6 +8,7 @@ import {
   CLOUD_BACKUP_ERRORS,
   isCloudBackupAvailable,
 } from '../handlers/cloudBackup';
+import { delay } from '../helpers/utilities';
```

Comment:
```suggestion
import { delay } from '@rainbow-me/helpers';
```

i think?

Reply:
There's no index file on helpers so it has to be `import { delay } from '@rainbow-me/helpers/utilities';`